src/frontend/app/page.tsx

```diff
@@ -11,10 +10,9 @@ export default function Home() {
       <Sidebar />
       <main className="flex flex-col flex-1">
         <ChatHeader />
-        <div className="flex-1 overflow-y-auto bg-background">
+        <div className="flex-1 overflow-hidden bg-background">
           {chatSelected ? <WelcomeSection /> : <ChatSection />}
         </div>
-        <ChatBottomBar />
```

Comment:
The `BottomBar` enables two interactions here:
1. Use the chat context to reply to messages.
2. If no chat is selected then start a new chat.

So if you refactor it to the chat section then please note that you will need to add another `ChatBottomBar` to the welcome section that starts a new chat if that is the design you are going for.